data/ecosystem/vendors.yaml

```diff
@@ -216,6 +216,13 @@
   contact: ''
   oss: true
   commercial: true
+- name: OpenObserve
+  distribution: false
+  nativeOTLP: true
+  url: 'https://openobserve.ai'
```

Comment:
Please link to your documentation where users can learn how they can use openobserve with OTLP